policy/Object.yaml

```diff
@@ -0,0 +1,392 @@
+    # decErr: not available yet DM-15180
+    refBand: # reference band is same for all measurements
+        functor: ReferenceBand
+        dataset: ref
+    # psRadecTai: No moving point source model yet.
+    # psRadec: No moving point source model yet.
+    # psPm: No moving point source model yet.
+    # psParallax: No moving point source model yet.
+    PsFlux:
+        functor: NanoJansky
+        dataset: forced_src  # recommend meas too
+        args: slot_PsfFlux_instFlux
+    PsFluxErr:
+        functor: NanoJanskyErr
+        dataset: forced_src
+        args:
+            - slot_PsfFlux_instFlux
+            - slot_PsfFlux_instFluxErr
+    # Assume the database will provide UDFs for Mags.
+    # psCov:
+    #     Flux should be somewhere in the name?
+    #     How best to store cov matrix?
+    # psLnL:  No moving point source model yet.
+    # psChi2:  No moving point source model yet.
+    # psNdata: No moving point source model yet.
+
+    # DRP is working the the Bulge-Disk model fluxes
+    # In the meantime, CModel is the best
+    # CModel fluxes have alias slot_ModelFlux
+    #bdRadec: # Same as SdssCentroid
+    BdE1:
+        functor: e1
+        dataset: meas
+        args:
+            - modelfit_CModel_ellipse_xx
+            - modelfit_CModel_ellipse_xy
+            - modelfit_CModel_ellipse_yy
+    # rBdE1Err:  Not computed
+    BdE2:
+        functor: e2
+        dataset: meas
+        args:
+            - modelfit_CModel_ellipse_xx
+            - modelfit_CModel_ellipse_xy
+            - modelfit_CModel_ellipse_yy
+    # rBdE2Err:  Not computed
+    BdReB:
+        functor: RadiusFromQuadrupole
+        dataset: meas
+        args:
+            - modelfit_CModel_dev_ellipse_xx
+            - modelfit_CModel_dev_ellipse_xy
+            - modelfit_CModel_dev_ellipse_yy
+    BdReD:
+        functor: RadiusFromQuadrupole
+        dataset: meas
+        args:
+            - modelfit_CModel_exp_ellipse_xx
+            - modelfit_CModel_exp_ellipse_xy
+            - modelfit_CModel_exp_ellipse_yy
+    #bdCov: Not computed
+    BdChi2:
+        functor: Column
+        dataset: meas
+        args: modelfit_CModel_objective
+    #bdSamples: Not computed
+    BdFluxB:
+        functor: NanoJansky
+        dataset: meas
+        args: modelfit_CModel_dev_instFlux
+    BdFluxBErr:
+        functor: NanoJanskyErr
+        dataset: meas
+        args:
+            - modelfit_CModel_dev_instFlux
+            - modelfit_CModel_dev_instFluxErr
+    BdFluxD:
+        functor: NanoJansky
+        dataset: meas
+        args: modelfit_CModel_exp_instFlux
+    BdFluxDErr:
+        functor: NanoJanskyErr
+        dataset: meas
+        args:
+            - modelfit_CModel_exp_instFlux
+            - modelfit_CModel_exp_instFluxErr
+    grStd:
+        # HSC users still debating
+        #  1) PSF convolved undeblended aperture fluxes?
+        #  2) Scarlet outputs?
+        # In the meantime: forced CModel
+        functor: Color
```

Comment:
Is this a Std from a given locus?  Is the function to compute that really just called “Color”?